lib/CtrlNest/Controller/Auth.pm

```diff
@@ -38,28 +71,47 @@ sub lockscreen {
 
   # Render template "auth/lockscreen.html.ep"
   return $self->render(
-    layout      => 'auth',
-    title       => 'Lockscreen',
-    page_layout => 'lockscreen',
-    username    => $username
+    layout   => 'auth',
+    title    => 'Lockscreen',
+    username => $username,
```

Comment:
Let's add a constant value in case the session does not exist. I'm thinking `SESSION_NOT_FOUND = 'session not found';`